test/policy/test_base.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
```

Comment:
I propose the tests to have the same structure and names as the src, makes finding the tests for a module very simple. This would be the start of it. WDYT?